src/components/Header/constants/config.js

```diff
@@ -0,0 +1,48 @@
+import search from '../../../images/search.svg';
+import offers from '../../../images/offer.svg';
+import help from '../../../images/help.svg';
+import signIn from '../../../images/signin.svg';
+import cart from '../../../images/cart.svg';
+
+export const headerLinksConfig = [
```

Comment:
you can have two different files for both headerlinks and breadcrumb and export it by default

Reply:
Yeah, i will change it